app/components/icons/WrapIcon.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import IconHug from "./IconHug";
+import type { Icon } from "../shared/icons-map";
+import iconsMap from "../shared/icons-map";
+
+export interface WrapIconProps {
+  icon?: Icon;
+  enableWrap?: boolean;
+}
+const WrapIcon = React.forwardRef<HTMLElement, WrapIconProps>(
```

Comment:
Rename the component to simply `Icon`

Reply:
Replaced Icon as IconType, as we were seeing some errors while import the component with the same name as that of Type